webapp/components/interop-dashboard.js

```diff
@@ -630,6 +634,26 @@ class InteropDashboard extends PolymerElement {
     return this.getYearProp('focusAreas')[name][prop];
   }
 
+  // Add the stable or experimental label to a tests URL depending on the view.
+  formatTestsURL(testsURL, stable) {
+    // Don't try to add a label if the URL is undefined or empty.
+    if (!testsURL) {
+      return testsURL;
+    }
+
+    if (stable) {
+      testsURL += '&label=stable';
```

Comment:
I would assume that "?" delimiter is always present in the testURL, e.g.   /results/ vs /results/? /results/&label=stable would fail here

Reply:
Yes, these are always wpt.fyi paths to test results pages with an interop label, so the query string should always already be present

Reply:
After trying this, I think using the URL object might not work in our specific case because we allow for multiples of the same query string parameter (e.g. multiple "label" params). Using the `set` method might overwrite a `&label=master` search param.

Reply:
I looked at the URLSearchParams [API](https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams).
Yeah, you're right, I wish there was a setAll method.

Here's a [fiddle](https://jsfiddle.net/m8zx7c1r/4/) with it working but it adds a lot of lines of code. (Probably can be optimized)

Reply:
I would update the implementation to match. I like @foolip's suggestion because search parameter modification shouldn't be something we deal with manually if we can avoid it. 
I did some searching to make sure we weren't the only ones looking for this same solution.


You should put a TODO note that it could be simplified and add these two links:
- https://github.com/whatwg/url/issues/762
- https://github.com/whatwg/url/issues/461